moduleroot/Rakefile

```diff
@@ -32,4 +32,4 @@ KafoModuleLint::RakeTask.new do |config|
   config.pattern = <%= @configs['param_docs_pattern'].inspect %>
 end
 
-task :default => [:validate, :lint, :spec]
+task :default => [:validate, :lint, :parallel_spec]
```

Comment:
puppetlabs_spec_helper 2.0.0 has a `release_checks` task that calls validate, lint, parallel_spec, plus additional release-readiness tests which would be best to use here.